src/NuGet.Protocol.Catalog/SimpleHttpClient.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class SimpleHttpClient : ISimpleHttpClient
+    {
+        private static readonly JsonSerializer JsonSerializer = CatalogJsonSerialization.Serializer;
+        private readonly HttpClient _httpClient;
+        private readonly ILogger _logger;
+
+        public SimpleHttpClient(HttpClient httpClient, ILogger<SimpleHttpClient> logger)
+            : this(httpClient, (ILogger)logger)
+        {
+        }
+
+        internal SimpleHttpClient(HttpClient httpClient, ILogger logger)
```

Comment:
>SimpleHttpClient [](start = 17, length = 16)

What's this constructor for?

Reply:
Catalog client's existing ctor (to avoid API breaking changes).